### ISSUE TITLE
Problem with textures after close interstitial ads

### ISSUE THREAD

**Issue by EmanuelOzorio** [Shin-NiL/Godot-Android-Admob-Plugin#105]
**Godot version:** 3.2.3 stable

**AdMob Plugin version:** 4.1.1

**Issue description:**
When I put `get_tree().change_scene("myscene.tscn")` on `_interstitial_closed` event the next scene opens without the textures.
Obs.: Opening the scene without ads works fine. The problemn only ocours with ads.

**Comment by gamergamesz** [comment]
Yup, the workaround is - preloaded the scene, pause , show ad, and on close ad resume game

**Comment by ienachescu** [comment]
Hey, I have a similar problem. I'm playing a reward ad and after it I'm changing a texture to a sprite, but the texture is black and shortly after that the game is crashing. I tried the pause workaround, but without success. Is there another workaround?

**Comment by gamergamesz** [comment]
That's means there is something wrong with your code maybe a typo, it usually happens due to that

**Comment by ienachescu** [comment]
I'm not sure about that, because if I'm not running the ad the texture appears

**Comment by ienachescu** [comment]
I resolved the texture problem by preloading the texture, but the game still crashes. In the game I have 2 buttons, one button sells the player an item for a price, and the other button sells the player an item for a lower price, but the player has to watch an ad. So basically when the player is rewarded by the ad the item is instanced. Maybe I'm doing something wrong? Because when I press the button that doesn't play the ad the game is not crashing and the code is pretty much the same on both buttons.

**Comment by ienachescu** [comment]
I found an workaround, I called get_tree().reload_current_scene() and the game stopped crashing.
